### ISSUE TITLE
SkShader custom build step will fail if shader file is not in Assets root folder

### ISSUE THREAD

**Issue by llfab** [StereoKit/StereoKit#329]
## Description
If the shader file (e.g. floor.hlsl) is not placed in the Assets root folder, an error message appears on the first build: no able to copy floor.hlsl.sks because file not found. Only for the 2nd build this would work. If you delete the app bin folder, this happens again on the first build

## Platform / Environment
Windows, Visual Studio 2022, SK for UWP

## Project file
```
    <!-- working -->
    <SKShader Include="Assets\floor.hlsl">
         <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
    </SKShader>

    <!-- NOT working-->
    <SKShader Include="Assets\Shaders\floor.hlsl">
         <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
    </SKShader>
```
As I analyzed this seem only fixable if you place the shader file into the Asset root folder. Earlier I wanted to put it in Assets\Shaders but if put there the above error in the first build would appear.

I suppose this may have to do with the custom build step in SKShader and its limitation on subfolder handling. 

Of course, this is not a big issue, but once you create a larger project you may want to organize your assets in different sub folders.

**Comment by maluoi** [comment]
The core issue here is likely this:
https://github.com/StereoKit/sk_gpu/issues/19

**Comment by maluoi** [comment]
Just verified that https://github.com/StereoKit/sk_gpu/issues/19 did fix the issue, and it's good to go in v0.3.6+ :)
